.dialyzer_ignore.exs

```diff
@@ -0,0 +1,4 @@
+[
```

Comment:
Can you describe why we need this ?

Reply:
@pankaj-ag We are running these actions in (MIX_ENV=test) and dialyzer also recognizes Elixir format of the ignore file and claim that those functions are errors. So, we are just ignoring it.

See [dialyzer readme](https://github.com/jeremyjh/dialyxir#elixir-term-format)